AbbaGarima/AG00001.xml

```diff
@@ -990,7 +994,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="NV" when="2019-10-21">Created stub</change>
+            <change who="NV" when="2019-10-21">Created an entry for Sergey Kim</change>
```

Comment:
Did you really create an entry or just added a reference?

Reply:
Changes in Zotero are not related to Bm xml records

Reply:
If this is the first time this Zotero tag is being used it may be a good occasion to correct it there and here. (Capitalize keyword)

Reply:
> Did you really create an entry or just added a reference?

my point was to explain that I created the entry for Sergey Kim, I used "entry" as a synonym for "stub", but now I changed it to "stub" since it might be confusing.

Reply:
Which entry or stub included in the record  AG00001 di you create and where?

Change is about revising this record, not other things.